src/Engine/Rendering/Renderer.cpp

```diff
@@ -193,7 +193,7 @@ Renderer::PickingResult Renderer::doPickingNow( const PickingQuery& query,
     if ( query.m_screenCoords.x() < 0 || query.m_screenCoords.x() > m_width - 1 ||
          query.m_screenCoords.y() < 0 || query.m_screenCoords.y() > m_height - 1 )
     {
-        result.m_roIdx = -1;
+        result.setRoIdx( Ra::Core::Utils::Index::Invalid() );
```

Comment:
```suggestion
        result.clear(); // reset to empty state
```

Reply:
Actually, we can simply return result, or {} since nothing is set from line 190 (decl) to here.

Reply:
Ok return {} sounds good.
There is another line in the code that looks the same.